.evergreen/.evg.yml

```diff
@@ -794,15 +794,7 @@ functions:
         shell: bash
         script: |
           ${PREPARE_SHELL}
-          cd "$MONGO_ORCHESTRATION_HOME"
-          # source the mongo-orchestration virtualenv if it exists
-          if [ -f venv/bin/activate ]; then
-            . venv/bin/activate
-          elif [ -f venv/Scripts/activate ]; then
-            . venv/Scripts/activate
-          fi
-          mongo-orchestration stop || true
-          cd -
+          sh ${DRIVERS_TOOLS}/.evergreen/stop-orchestration.sh || true
```

Comment:
We use `bash` everywhere else.  Should we do the same here?

Reply:
```suggestion
          bash ${DRIVERS_TOOLS}/.evergreen/stop-orchestration.sh || true
```

Reply:
Done!